mingw-w64-git/PKGBUILD

```diff
@@ -177,14 +185,18 @@ build() {
   done &&
 
   case "${pkgname[@]}" in *pdb*)
-    make -f ../mingw-w64-git.mak STRIP=$PWD/../cv2pdb-strip STRIP_OPTS= strip-all;;
+    # TODO Clang already creates .pdb files for us, but should we also strip files with llvm-strip?
+    [[ "$MSYSTEM" == "CLANGARM64" ]] || make -f ../mingw-w64-git.mak STRIP=$PWD/../cv2pdb-strip STRIP_OPTS= strip-all;;
```

Comment:
Could we maybe change this so that we always invoke the `make strip-all` command, but override `STRIP` and `STRIP_OPTS` in ways specific to clang and non-clang, respectively?